src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header class="header">
+        <a href="#">
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
```

Comment:
The `data-qa` attribute should be `nav-hover` for the fourth navigation link (`Laptops & Computers`) as specified in the task requirements.